### DISCUSSION TITLE
Can I add per-project snippets?

### DISCUSSION THREAD

**Discussion by gsinclair** [dcampos/nvim-snippy#93] [Q&A] [answered]
I would like to have a file `local.snippets` for example and source it in `local-vimrc.vim`, to contain snippets that only apply to this project. I don't particularly care about filetype in this instance, but I would be happy to specify the filetype(s) as well. My project contains Latex and Lua, and both types of file would benefit from some local snippets.

Or, I could cut out the middleman and just define some snippets in `local-vimrc.vim` directly, if I knew how.

Is there a way?

**Comment by dcampos** [comment, marked as answer]
There was no easy way to do that. I've just pushed a commit that allows putting project-specific snippets in a local directory, by default called `.snippets`. The local directory's relative path can be configured, and it can even be an absolute path. Check the docs for how to configure it.

> Or, I could cut out the middleman and just define some snippets in local-vimrc.vim directly, if I knew how.

I can't think of a way to achieve that currently, but it sure would be a nice feature to be added.
